### PR TITLE
chore: remove deprecated components FE-1921

### PR DESCRIPTION
**Pull request by jamime**
### Proposed behaviour

• remove deprecated simple-color-picker
• remove deprecated radio-button
• remove deprecated fieldset
• remove deprecated date-range
• remove deprecated date
• remove deprecated dropdown-filter-ajax
• remove deprecated dropdown-filter
• remove deprecated create

### Current behaviour

We have an experimental and a deprecated version.

### Checklist

<del>- [ ] Screenshots are included in the PR
<del>- [ ] Carbon implementation and Design System documentation are congruent
<del>- [ ] All themes are supported
- [x] Commits follow our style guide
- [x] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [x] Storybook added or updated
- [x] Typescript `d.ts` file added or updated

### Additional context
This is a long lived branch. Merging this PR will not trigger a release.

### Testing instructions
Ensure that there are no regressions in storybook, all non-deprecated functionality should be unaffected.

### CODE REVIEW

File: src/utils/decorators/input/__spec__.js [hunk 1]
```diff
@@ -32,6 +32,29 @@ class TestClassTwo extends React.Component {
 
   get inputProps() {
     let { ...props } = this.props;
+    console.log(this.props)
```
Comment (edleeks87) [inline src/utils/decorators/input/__spec__.js:35]: Is it intentional to leave the console.log statements in?

File: src/utils/decorators/input/__spec__.js [hunk 2]
```diff
@@ -32,6 +32,29 @@ class TestClassTwo extends React.Component {
...
+
+class TestClassThree extends React.Component {
+  count = 0;
+
+  shouldComponentUpdate(nextProps, nextState) {
+    this.count++;
+  }
+
+  handleChange() {
+    console.log('test')
```
Comment (edleeks87) [inline src/utils/decorators/input/__spec__.js:52]: See above

**Comment by jamime** [comment]
This seems fine to me, but I can't approve it as I created the PR.

**Comment by sianford** [comment]
Tested in Safari browser only. All required components have been removed as required. All others work as expected. Testing passed ✅

**Pull request merged**